POKEMON_OF_THE_DAY/env.default

```diff
@@ -0,0 +1,17 @@
+# Information in the .env is used during a local run to mock the environment provided when the integration is run on Pandium
```

Comment:
maybe include a note indicating that this should be copied to `.env` and filled out there, rather than filling in this file.
also, I'm not sure if the `.env should be in .gitignore` note is necessary since this is not an action required of the user ... the `.gitignore` is in source control. or it could say `.env is included in the .gitignore...`

Reply:
I'm not sure what you mean when you say ".gitignore is in source control" but if I take .env out of my .gitignore and push my code the .env will end up on github (you can see it in this commit: 
https://github.com/pandium/sample_integrations/pull/7/commits/495b3f6dcbd13608a720faa8c3699397b72df360) and we don't want that

Reply:
All I meant was that the wording was confusing to me, since it sounded like we are directing the user to add `.env` to the `.gitignore` file, however this action is already completed